commcare_connect/opportunity/management/commands/populate_completed_work_user_visits.py

```diff
@@ -0,0 +1,30 @@
+from itertools import groupby
+
+from django.core.management.base import BaseCommand
+
+from commcare_connect.opportunity.models import CompletedWork, PaymentUnit, UserVisit
+
+
+class Command(BaseCommand):
+    help = "Populates location for user visits from form json"
+
+    def add_arguments(self, parser, *args, **kwargs):
+        parser.add_argument("--opp", type=int)
+
+    def handle(self, *args, **options):
+        opp_id = options.get("opp")
+        filter_kwargs = {"opportunity": opp_id} if opp_id else {}
+        payment_units = PaymentUnit.objects.filter(**filter_kwargs)
+        for payment_unit in payment_units:
+            filter_kwargs.update(deliver_unit__in=payment_unit.deliver_units.values_list("id", flat=True))
+            user_visits = UserVisit.objects.filter(**filter_kwargs)
```

Comment:
this is sketchy without user filtering since it will associate forms from different users into one unit of work